src/Controls/src/Core/BindableObject.cs

```diff
@@ -589,7 +589,24 @@ BindablePropertyContext CreateAndAddContext(BindableProperty property)
 		}
 
 		[MethodImpl(MethodImplOptions.AggressiveInlining)]
-		internal BindablePropertyContext GetContext(BindableProperty property) => _properties.TryGetValue(property, out var result) ? result : null;
+		internal BindablePropertyContext GetContext(BindableProperty property)
+		{
+			var isValueFound = _properties.TryGetValue(property, out var result);
+
+			if (isValueFound)
+				return result;
+
+			// Some properties may not be found due to the dictionary's BindableProperty class key
```

Comment:
So I added this here because it seemed that the 'IsClippedToBounds' property that I was looking for inside the _properties dictionary did exist in there, but it was not registered in the TryGetValue call.

The _properties dictionary uses the class BindableProperty for the key and *from what I can tell* it was not finding the property for some reason. I thought about overriding the GetHashCode and Equals methods in the BindableProperty Class but that would be a public facing change and I was not 100% sure if that is what is going on here.

Please let me know if this change is not wanted!

An example of properties found in this foreach loop that were not found by `_properties.TryGetValue(property, out var result)`: 
* ImplicitStyle
* Text
* IsClippedToBounds

Reply:
Looks like this caused some other tests to fail so I will just apply it to the IsClippedToBounds property / explore this a little more